README.md

```diff
@@ -167,12 +167,12 @@ Assuming tn1.json is a testnet config file saved in the same directory as go-spa
 3. Stop go-spacemesh and start it with the following params:
 
 ```bash
-./go-spacemesh --grpc-server --json-server --tcp-port [a_port] --config [configFileLocation] -d [nodeDataFilesPath] --coinbase [coinbase_account] --start-mining --post-datadir [dir_for_post_data]
+./go-spacemesh --tcp-port [a_port] --config [configFileLocation] -d [nodeDataFilesPath] --coinbase [coinbase_account] --start-mining --post-datadir [dir_for_post_data]
```

Comment:
we don't need GRPC servers anymore?

Reply:
It's necessary if you're going to connect smapp or the CLIWallet to it, but it's not necessary if you just want to run a node on the command line, which is what these instructions are for. We still need to separately document the API (see #2083) since it's a bit confusing how to use it properly.